src/main/java/com/openshift/internal/restclient/model/Container.java

```diff
@@ -136,11 +139,31 @@ public void setPorts(Set<IPort> ports) {
 	public void setImagePullPolicy(String policy) {
 		set(node, propertyKeys, IMAGE_PULL_POLICY, policy);
 	}
-
+	
 	@Override
 	public String getImagePullPolicy() {
 		return asString(node, propertyKeys, IMAGE_PULL_POLICY);
 	}
+	
+	@Override
+    public void setCommand(Set<String> command) {
```

Comment:
from what i understand, command is equivalent to DockerFile EntryPoint and args is equivalent to dockerfile CMD.  Both are list.